hie.yaml.stack

```diff
@@ -6,6 +6,14 @@ cradle:
     - path: "./test/testdata/"
       config: { cradle: { none:  } }
 
+    - path: ./Setup.hs
+      config:
+        cradle:
+          direct:
+            arguments:
+              - "-package Cabal"
+              - "-package base"
```

Comment:
I am not sure this works for correctly stack. Will the correct cradle be shown? Almost definitely not, it will be a multi-cradle from now on, instead of Stack. So this might kill the hls-wrapper, since it decides based on the cradle type the ghc version.

Reply:
Mmm, maybe was you thinking in hie instead hls?

About stack and cradle, the issue is the direct cradle assumes you have ghc on path and that is not the common case for stack. We could remove the subcradle or add a comment, cause the load of setup will fail anyway, but with a diff error.

Reply:
> Mmm, maybe were you thinking in hie instead hls?

I was rather thinking of a work-arounds for experts. You can make it work if you really know what you are doing, but I dont think this works for average case. E.g. since this will longer be identified as a stack cradle, rather a multi-cradle, hls-wrapper wont do what we expect.

We probably need more support for it via hie-bios.